emulated_hue/config.py

```diff
@@ -16,7 +16,7 @@
 class Config:
     """Hold configuration variables for the emulated hue bridge."""
 
-    def __init__(self, hue, data_path, hass_url, hass_token):
+    def __init__(self, hue, data_path, hass_url, hass_token, ip_address, mac_address):
```

Comment:
Maybe have these as optional params by setting default value to None.
In the check below, use a check for None to use the previous call to get the ip

Reply:
Hmm yes good idea. Although they are currently "optional" as they are defaulted to None by the argument parser. What do you think is better? Feel free to commit changes.

Reply:
The parses is not used when we're called as library, only when called as script (__main__.py) so best practice to make sure the API doesn't break.

Reply:
Ah I guess I didn't see that it would be used as a library, but I suppose I could add that. Doesn't hurt.